src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,46 @@
+package lotto.domain;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoResult {
+    private List<Ranking> rankingList;
+
+    public LottoResult() {
+        rankingList = new ArrayList<>();
+    }
+
+    public LottoResult(List<Ranking> rankingList) {
+        this.rankingList = rankingList;
+    }
+
+    public List<Ranking> findRankings(int matchingCount) {
+        List<Ranking> result = new ArrayList<>();
+        Ranking target = Ranking.findRank(matchingCount);
+        for (Ranking ranking : rankingList) {
+            addRankingWhenSame(result, ranking, target);
+        }
+        return result;
```

Comment:
변수 선언이 다소 복잡해서 아래 처럼 `stream`을 활용해볼수 있을것 같아요.

`stream`이 항상 가독성을 좋게하는건 아니지만 적절한 활용한다면 좋다고 생각해요~ :)
```suggestion
        Ranking target = Ranking.findRank(matchingCount);
        return rankingList.stream()
            .filter(ranking -> ranking.equals(target))
            .collect(Collectors.toList());
```

Reply:
샘플 코드까지 작성해주셨네요.  유용하게 활용할 수 있는 부분에 대해서는 사용하는 연습을 하겠습니다! 😄